File: app/src/main/java/com/luizroseiro/testeandroidv2/models/ErrorResponse.java
package com.luizroseiro.testeandroidv2.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ErrorResponse {

    @SerializedName("code")
    @Expose
    private String code;

    @SerializedName("message")
    @Expose
    private String message;

    public String getCode() {
        return code;
    }

    public String getMessage() {
        return message;
    }

}

File: App/data/src/main/java/com/santander/data/source/remote/entity/response/ErrorResponse.kt
package com.santander.data.source.remote.entity.response

import com.google.gson.annotations.SerializedName

data class ErrorResponse(

	@field:SerializedName("code")
	val code: Int? = null,

	@field:SerializedName("message")
	val message: String? = null
)